### ISSUE TITLE
An alias can not reference itself, got a circular reference on ProductModel

### ISSUE THREAD

**Issue by skyhell59** [pomm-project/pomm-bundle#96]
Hi, I have a problem since I put version 2.4. My models declarent as Symfony services like that : 
```
    App\Infrastructure\Model\ProductModel:
        public: true
        tags: [pomm.model]
```
I have an error when used my model : `An alias can not reference itself, got a circular reference on ProductModel`

I tried to delete `tags: [pomm.model]` but a new error has occured : 

`ProductModel is not registered against the session`

**Comment by sanpii** [comment]
It seems there is a bug when the name of the service is the class name.

You can write:

```yaml
    product_model:
        class: App\Infrastructure\Model\ProductModel:
        public: true
        tags: [pomm.model]
```

**Comment by sanpii** [comment]
Fixed in [2.4.1](https://github.com/pomm-project/pomm-bundle/releases/tag/2.4.1), thank you.

**Comment by skyhell59** [comment]
Thanks :+1:
